guides/common/modules/ref_supported-browsers.adoc

```diff
@@ -20,7 +20,7 @@ Other browsers may work unpredictably.
 endif::[]
 
 ifdef::satellite[]
-The {ProjectWebUI} and command-line interface support English, Simplified Chinese, Japanese, French.
+The {ProjectWebUI} and command-line interface support English, Simplified Chinese, Japanese, French, and Korean.
```

Comment:
Should this be ordered alphabetically?

Reply:
In a way, I'm just repaying the favor from https://github.com/theforeman/foreman-documentation/pull/2278#discussion_r1266579816.